.internal-ci/test/minting-config-tx-test.sh

```diff
@@ -66,13 +57,4 @@ mc-consensus-mint-client generate-and-submit-mint-config-tx \
     --total-mint-limit 10000000000
 
 echo "-- sleep and wait for tx/blocks to sync"
-
-new_block_count=0
-echo "-- Waiting for mint config tx to commit to the block chain"
-
-while [[ $block_count -ge $new_block_count ]]
-do
-    sleep 15
-    new_block_count=$(get_block_count)
-    echo "  Current block count: $new_block_count"
-done
+sleep 15
```

Comment:
:question: Did the logic to wait for the blocks mean to get removed?

Reply:
Good catch